src/html-to-govspeak.js

```diff
@@ -232,54 +232,64 @@ function getWordListLevel (node) {
   return levelMatch ? parseInt(levelMatch[1], 10) : undefined
 }
 
-function isWordListItem (node) {
-  return !!getWordListLevel(node)
+function isMsWordListItem (node) {
+  return !!getMsWordListLevel(node)
 }
 
-service.addRule('addWordListItem', {
-  filter: (node) => isWordListItem(node),
+// Based on a node that is a list item in a MS Word document, this returns
+// the marker for the list.
+function msWordListMarker (node, bulletListMarker) {
+  const markerElement = node.querySelector('span[style="mso-list:Ignore"]')
+
+  // assume the presence of a period in a marker is an indicator of an
+  // ordered list
+  if (!markerElement || !markerElement.textContent.match(/\./)) {
+    return bulletListMarker
+  }
+
+  const nodeLevel = getMsWordListLevel(node)
+
+  let item = 1
+  let potentialListItem = node.previousElementSibling
+
+  // loop through previous siblings to count list items
+  while (potentialListItem) {
+    let itemLevel = getMsWordListLevel(potentialListItem)
+
+    // if there are no more list items or we encounter the lists parent
+    // we don't need to count further
+    if (!itemLevel || itemLevel < nodeLevel) {
```

Comment:
I'd find this a little clearer if we used the names `itemLevel` and `previousItemLevel`.

Reply:
Sorry for the delay - this slipped my mind, it's not actually the previous item level. It's the level of the node that we're determining the marker for.